docs/development/CI/cscs-ci.md

```diff
@@ -0,0 +1,35 @@
+# CSCS-CI Configuration
+
+## Initial Setup
+
+Follow the steps in the [CSCS-CI documentation](https://gitlab.com/cscs-ci/ci-testing/containerised_ci_doc). You will require the help of someone who can register the project to be allowed to run CI on CSCS machines.
+
+## Current Configuration
+
+The configuration can be viewed and changed on the [setup page](https://cicd-ext-mw.cscs.ch/ci/setup_ci) after logging in with the credentials for "cscs-ci setup for gt4py", which can be found on the internal credentials store in the GT4Py folder.
+
```

Comment:
Shouldn't we add some info about the current workflow file (`ci/cscs-ci.yml`)? At least a mention to the current location and the general structure (maybe even a link to basic information about gitlab workflows syntax).